.github/workflows/github-automerge.yaml

```
@@ -0,0 +1,148 @@
+name: github-automerge
+
+on:
+  workflow_call:
+    inputs:
+      PRIMUS_REF:
+        description: 'The primus ref to checkout.'
+        required: true
+        default: 'main'
```

actionlint (1.7.10)

[error] input "PRIMUS_REF" of workflow_call event has the default value "main", but it is also required. if an input is marked as required, its default value will never be used

(events)